### ISSUE TITLE
Group Improvements

### ISSUE THREAD

**Issue by vmwinckler** [koraktor/steam-condenser#225]
**Suggest add in group data:** (return from SteamGroup)
- Group Name
- Group Abreviate Name
- Admin Member Mark

My return:

``` PHP
SteamGroup Object
(
    [customUrl:SteamGroup:private] => 
    [fetchTime:SteamGroup:private] => 1379436150
    [groupId64:SteamGroup:private] => 103582791432487214
    [members:SteamGroup:private] => Array
[...]
```

**Comment by koraktor** [comment]
:+1: 

Didn't know that XML data for groups does now offer a bit more information.

But I'm not quite sure what you mean by "Admin Member Mark".

**Comment by vmwinckler** [comment]
Actually I do not know if the XML features, I found this information searching directly in HTML but I'm horrible and slow.

About "Admin Member Mark": would be only one brand for members who are administrators of the group, another alternative would be a second array containing only members administrators

**Comment by koraktor** [comment]
The other information seems to be fine to add, but the information about who's admin is not available through an API.

**Comment by vmwinckler** [comment]
Over administrators, i find in HTML class "rank_icon". i belive i can find admins and other officers by "rank".

today dont return rank(return with "fetch default"):

``` PHP
[members:SteamGroup:private] => Array
        (
            [0] => SteamId Object
                (
                    [customUrl:SteamId:private] => 
                    [fetchTime:SteamId:private] => 
                    [friends:SteamId:private] => 
                    [games:SteamId:private] => 
                    [limited:SteamId:private] => 
                    [nickname:SteamId:private] => 
                    [playtimes:SteamId:private] => 
                    [steamId64:SteamId:private] => 76561198012606481
                    [tradeBanState:SteamId:private] => 
                )
```

**Comment by koraktor** [comment]
This is now implemented in the `1.3-stable` branches of each implementation.
